sap/5.9/modules/ROOT/pages/sap-connector-config-snc.adoc

```diff
@@ -51,6 +52,54 @@ You can use STRUST to see names that are already in use.
 +
 `sapgenpse <export_own_cert> –v –p RFC –o RFC_MULE.crt`
 
+[[configure-the-client-environment-when-deploying-in-cloudhub-2]]
+== Configure the Client Environment When Deploying in CloudHub 2.0
+
+In SAP Connector 5.9.5 and later, an application that uses the Certificate connection type can be deployed in CloudHub 2.0. SAP Connector has an embedded installer that sets up the Personal Security Environment (PSE).
+
+You must first create your application:
+
+. Open Anypoint Studio.
+. Navigate to the `src/main/resources` folder of your application.
+. In the `src/main/resources` folder, create a folder named `sec`.
++
+image::sap-resources-sec.png[Sap Resources Sec Folder]
++
+. Add the following files in the `sec` folder:
+* `libsapcrypto.so`
++
+The Linux version of the SAP CommonCryptoLib.
+* `sapgenpse`
++
+The Linux version of the SAP command line tool that manages PSE files.
+* `RFC.pse`
++
+The file you created in the <<configure-the-client-environment, previous step>>.
+. Because CloudHub 2.0 runs on Linux machines, ensure that any dependency of the application is compiled for Linux, including `com.sap.conn.jco:libsapjco3`.
+. Save the application and export the `jar` file that is used for CloudHub 2.0 deployment.
+
+You must then import the created application in CloudHub 2.0:
+
+. Log in to your Anypoint Platform account and navigate to Runtime Manager.
+. Click on *Deploy Application*.
+. In *Application Name*, enter a name for the application, for example, `cert-ch2`.
+. In *Deployment Target*, select `CloudHub 2.0` if it is not selected already.
+. Click *Choose File* and upload your `jar` file.
+. In the *Properties* tab, add the following:
++
+[%header,cols="30a,30,30"]
+|===
+|Property|Example Value|Description
```

Comment:
Property > Field

Reply:
We need to discuss how we punctuate field names in tables. To me, using bold or back ticks for this is too much and makes things less readable. The purpose of bold and back ticks is to have things stand out in text, which is not needed here. Anyway, if you still want to do this, these should probably be bold.

Reply:
Leaving the table header as Property because these are specifically properties, not fields